ui/raidboss/popup-text.ts

```diff
@@ -212,48 +209,40 @@ const textMap: TextMap = {
   },
 };
 
-// Helper for handling trigger overrides.
-//
-// asList will return a list of triggers in the same order as append was called, except:
-// If a later trigger has the same id as a previous trigger, it will replace the previous trigger
-// and appear in the same order that the previous trigger appeared.
-// e.g. a, b1, c, b2 (where b1 and b2 share the same id) yields [a, b2, c] as the final list.
-//
-// JavaScript dictionaries are *almost* ordered automatically as we would want,
-// but want to handle missing ids and integer ids (you shouldn't, but just in case).
-class OrderedTriggerList {
```

Comment:
What was wrong with OrderedTriggerList?

Reply:
Nothing, just it mix-up the localization step and override step, and not they are separated.

Reply:
I guess I don't really understand.  You could replace `ProcessedTrigger` with `LocalizedTrigger` and this code would still work.  Mostly I just know this existing code is working, and so I'd rather not replace it with something new, unless there's a reason to rewrite it.

Reply:
In particular, I think it's a little confusing from a user output error to have the override console messages come in the opposite order that they are overridden.  If you have A B C triggers in order, this will print out C overrides B, and B overrides A, but in reality the load order is A B C.